samples/net/wpan_serial/sample.yaml

```diff
@@ -5,5 +5,6 @@ tests:
   sample.net.wpan.serial:
     harness: net
     depends_on: usb_device
+    extra_args: OVERLAY_CONFIG="overlay-cc2520.conf"
     platform_whitelist: quark_se_c1000_devboard
```

Comment:
Is this really the only board or configuration where this can be run? Can you please remove the whitelisting and enable other boards for testing?

Reply:
This overlay only works on the boards with CC2520 radio, AFAIK it is only `quark_se_c1000_devboard`. There are boards where radio enables itself (if you select 802.15.4) through defconfigs like `reel_board`.